flow/Net2.actor.cpp

```diff
@@ -1500,8 +1504,27 @@ void ASIOReactor::sleep(double sleepTime) {
 				this->firstTimer.expires_from_now(boost::posix_time::microseconds(int64_t(sleepTime*1e6)));
 				this->firstTimer.async_wait(&nullWaitHandler);
 			}
-			setProfilingEnabled(0); // The following line generates false positives for slow task profiling
-			ios.run_one();
+			setProfilingEnabled(0); // Putting the thread to sleep generates false positives for slow task profiling
+			size_t count = 0;
+
+			if (FLOW_KNOBS->BUSY_WAIT_THRESHOLD != 0) {
+				// poll epoll_wait for a short time in the hope that an I/O will complete
+				// This is usually a good assumption (if we're running right now, we're probably not about
+				// to become completely idle), and it is faster than having Linux suspend and reschedule us.
+				double sleep_time = last_io + FLOW_KNOBS->BUSY_WAIT_THRESHOLD;
```

Comment:
This isn't universally adhered to, but generally variables in this codebase use camel case rather than underscores.

Reply:
Force pushed changes to the variable names (and nothing else).